package.json

```diff
@@ -1,34 +1,39 @@
 {
```

Comment:
> If you do a npm install --save-dev X or npm install --save Y it will autoformat the package.json to be 2 spaces.

Hm, TIL.  That's exactly what happened.